com.unity.render-pipelines.high-definition/Editor/RenderPipeline/HDRenderPipelineUI.cs

```diff
@@ -13,10 +13,9 @@ static partial class HDRenderPipelineUI
     {
         enum Expandable
         {
-            // CameraFrameSettings = 1 << 0, //obsolete
-            // BakedOrCustomProbeFrameSettings = 1 << 1, //obsolete
-            // RealtimeProbeFrameSettings = 1 << 2, //obsolete
-            // General = 1 << 3, // empty - settings moved to HD Global Settings
+            PunctualLightShadows = 1 << 1,
```

Comment:
@RSlysz is it safe to reuse? i would think not but i am probably wrong
i am guessing users may have used those values if they branched out HDRP

Reply:
we have so many foldouts here, so I wasn't able to store it in new ones

Reply:
Well, this is **reusable**. Up to the following limitation:
- We cannot ensure anymore editor state (expended/collapsed) while introducing the new scope. It is part of data that are saved inside the EditorPrefs.
- There is no possible migration to ensure state as this is a static data from EditorPrefs. At least, it is not possible to handle it inside the HDRenderPipelineAsset. We don't have any framework to deal with this kind of change.

Both seamed really minor to me but perhaps an UX can find it bothering.